v2/internal/testcommon/vcr/redact.go

```diff
@@ -109,6 +128,15 @@ func HideRecordingData(azureIDs creds.AzureIDs, s string) string {
 	return s
 }
 
+func (r *Redactor) HideRecordingDataWithCustomRedaction(s string) string {
```

Comment:
I'd make this internal, and just call it from all the existing entry points; that way we can guarantee these custom redactions are applied everywhere. It'd be unfortunate to redact something from the body of a request, but leave it present in headers, for example.

This would also mean consumers wouldn't need to change as they'd still be calling `HideRecordingData()`

Reply:
yep, sounds good. I've taken the suggestion and calling the `hideRecordingDataWithCustomRedaction` from other methods(HideURL, HideHeaders) as well